winui/Scheduler/Themes.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: post
+title: Themes in WinUI Scheduler control | Syncfusion
+description: Learn here all about to use various themes in Syncfusion WinUI `Scheduler`(`SfScheduler`) control and more.
+platform: winui
+control: SfScheduler
+documentation: ug
+---
+
+# Themes in WinUI Scheduler (SfScheduler)
+ The WinUI Scheduler allows you to apply different themes to customize its appearance for a better user exoerience.
+
+Refer to the following link for guidance on applying themes: [Themes for Syncfusion WinUI controls](https://help.syncfusion.com/winui/common/themes) 
+
```

Comment:
@VidyalakshmiMani , Update changes as we discussed through teams call.

Reply:
Updated the changes as discussed.